pkg/local_object_storage/engine/evacuate_test.go

```diff
@@ -28,7 +28,7 @@ func newEngineEvacuate(t *testing.T, shardNum int, objPerShard int) (*StorageEng
 
 	e := New(
 		WithLogger(zaptest.NewLogger(t)),
-		WithShardPoolSize(1))
+		WithShardPoolSize(uint32(objPerShard)))
```

Comment:
in general, doubtful fix to me possibly hiding buggy implementation or test

Reply:
> doubtful fix to me

why? this is _literally_ the problem, i proofed it locally, required 100k+ test runs. `ants.Pool` has background workers and some logic that blocks execution (calculates num of wip workers), if it does not free it before another iteration is being done, current shard logic just tries another shard and duplicates objects

Reply:
in other words, 1-sized pool cannot guarantee that an object will be put if you want to put 2+ objects. on a few-core machine, it is more critical, on my notebook it is like ~ 0.003%. but an evacuate test should not suffer because of the put problem IMO, it should test evacuation logic and it cannot, because objects are duplicated and go to unexpected shards